packages/dbml-parse/src/lib/utils.ts

```diff
@@ -3,9 +3,15 @@ import { SyntaxNode } from './parser/nodes';
 import { getTokenFullEnd, getTokenFullStart } from './lexer/utils';
```

Comment:
Please combine all of this into 1 test file.

Reply:
@PhucNguyen101 fixed